### PR TITLE
Avoid saving without changes

### PR DESCRIPTION
**Pull request by juliushaertl**
First attempt towards fixing #2337 

- [x] Test case for trailing node transaction triggering the step and save
- [x] Hotfix by appending a trailing paragraph before creating the tiptap document
  - [ ] Hopefully figure out a nicer way to solve this

### CODE REVIEW

File: src/EditorFactory.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ const createEditor = ({ content, onCreate, onUpdate, extensions, enableRichEditi
 	}
 	extensions = extensions || []
 	return new Editor({
-		content,
+		content: content + '<p/>',
```
Comment (juliushaertl) [inline src/EditorFactory.js:125]: I fail to come up with a nicer approach for this, any ideas or objections on using this @nextcloud/text ?

Comment (vinicius73) [reply src/EditorFactory.js:125]: I can't think about better solution than this.

**Comment by vinicius73** [comment]
the expects now have a `p` tag 🙈 

image

**Comment by juliushaertl** [comment]
@vinicius73 Which would be expected so that we always end up with a new line. But I didn't see a failing CI check. Anyways rebased, so let's see.

**Comment by max-nextcloud** [comment]
Ahh.... i see. Tests still need to be updated.

**Review by max-nextcloud — APPROVED** [review]
Not ideal but seems to do the trick.

**Comment by juliushaertl** [comment]
/backport cdfcd37e39067fb60b515d306251b0ec24776f70,2fa358f8e7bfdc74158fa1e3a8b352db81806f25,5c56006d74a95396eb8492c45ca17d544f7d7a5e to stable24

**Comment by juliushaertl** [comment]
/backport cdfcd37e39067fb60b515d306251b0ec24776f70,2fa358f8e7bfdc74158fa1e3a8b352db81806f25,5c56006d74a95396eb8492c45ca17d544f7d7a5e to stable23

**Pull request merged**